--- python/CMakeLists.txt       2024-05-15 16:58:18.757344399 -0500
+++ python/CMakeLists.txt       2024-05-15 16:58:47.277960416 -0500
@@ -38,11 +38,11 @@
 
 install(
   FILES ${CMAKE_CURRENT_BINARY_DIR}/uhdm.py
-  DESTINATION ${CMAKE_INSTALL_PREFIX}/${Python_site_packages}
+  DESTINATION ${Python_site_packages}
 )
 install(
-  TARGETS ${SWIG_MODULE_py_udhm_REAL_NAME}
-  LIBRARY DESTINATION ${PYTHON_SITE_PACKAGES}
+  TARGETS py_uhdm
+  LIBRARY DESTINATION ${Python_site_packages}
 )
 
 add_test(

