diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 4ab5a78..4f3b68d 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -23,8 +23,13 @@ swig_add_library(py_uhdm
   ${PROJECT_NAME}.i swig_test.cpp
 )
 target_link_libraries(py_uhdm
-  PRIVATE uhdm ${Python3_LIBRARIES}
+  PRIVATE uhdm
 )
+IF(APPLE)
+    set_property(TARGET py_uhdm APPEND_STRING PROPERTY
+        LINK_FLAGS "-undefined dynamic_lookup")
+ENDIF()
+
 set_target_properties(py_uhdm PROPERTIES
   LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/
 )
